pynecone/utils/format.py

```diff
@@ -358,6 +361,14 @@ def format_dataframe_values(value: Type) -> List[Any]:
 
     return format_data
 
+def formant_image_data(value: Type) -> str:
```

Comment:
Is it format_image_data instead of formant_image_data?

Reply:
+1